proteus/postprocessing.c

```diff
@@ -3767,12 +4275,12 @@ void calculateConservationResidualPWL(int nElements_global,
 	 /*mwf debug
 	 printf("calcConsResPWL eN=%d nN=%d starR=%g\n",eN,nN,
 		starR[nN_global][eN_star]);
-	 */
+	 //	 */
```

Comment:
not a big deal, but I'd removed the mixed // and */ comments within the multiline /* */ comment